### PR TITLE
Javascript week1 homework

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**

### CODE REVIEW

File: javascript/javascript1/week1/main.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1996;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
+
```
Comment (vulk29) [inline javascript/javascript1/week1/main.js:6]: great start! I only have one comment here and throughout your code, you seem to use exclusively let. Would it make sense to use const here? where? :)

File: javascript/javascript1/week1/main.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+  dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+  console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture + ".");
+} else {
+  dogYear = dogYearFuture - dogYearOfBirth;
+  console.log("Your dog will be " + dogYear + " years old in " + dogYearFuture + ".");
+}
+
```
Comment (vulk29) [inline javascript/javascript1/week1/main.js:20]: Great job! nice and clean! :)

File: javascript/javascript1/week1/main.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  actualHousePrice =
+    houseWide[i] * houseHight[i] * houseDepth[i] * 2.5 * 1000 +
+    gardenSize[i] * 300;
+  if (actualHousePrice < housePrice[i]) {
+    console.log("Hey " + names[i] + ", sorry but You are paing too much for this house.");
+  } else {
+    console.log("Hey " + names[i] + ", the price for the house is OK.");
+  }
+}
+
```
Comment (vulk29) [inline javascript/javascript1/week1/main.js:40]: great use of arrays!!

File: javascript/javascript1/week1/main.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+// Ez Namey (Startup name generator)
+
+let firstWords = ["Easy", "Awesome", "Super", "Cool", "Great", "Nice", "Your", "Corporate", "First", "Most"];
+let secondWords = ["Thing", "Stuff", "Idea", "Solution", "Project", "Future", "Life", "Desicion", "Friend", "People"];
+let startupName = [];
+
+for (let y = 0; y < firstWords.length; y++) {
+  startupName.push(firstWords[Math.floor(Math.random() * 10)] + " " + secondWords[Math.floor(Math.random() * 10)]);
+  console.log( "The startup: " + startupName[y] + " contains " + startupName[y].length + " characters." );
+}
```
Comment (vulk29) [inline javascript/javascript1/week1/main.js:50]: I think you were only supposed to generate one :). Bonus points for the loop!

**Review by vulk29 — APPROVED** [review]

File: javascript/javascript1/week1/main.js (hunk 1 above)
Comment (oleksandra-lavryk) [reply javascript/javascript1/week1/main.js:6]: @vulk29 maybe it make sense to use const for date of birth it will not change :)

**Pull request merged**